_layouts/compress.html

```diff
@@ -1,5 +1,5 @@
 ---
-## 
+##
```

Comment:
right, here is where dev and prod matters; also we are removing these (as mentioned)